presto-main/src/main/java/com/facebook/presto/server/PrestoSystemRequirements.java

```diff
@@ -70,10 +70,10 @@ private static void verifyOsArchitecture()
         String osName = StandardSystemProperty.OS_NAME.value();
         String osArch = StandardSystemProperty.OS_ARCH.value();
         if ("Linux".equals(osName)) {
-            if (!"amd64".equals(osArch) && !"ppc64le".equals(osArch)) {
-                failRequirement("Presto requires amd64 or ppc64le on Linux (found %s)", osArch);
+            if (!"amd64".equals(osArch) && !"ppc64le".equals(osArch) && !"aarch64".equals(osArch)) {
+                failRequirement("Presto requires amd64 or ppc64le or aarch64 on Linux (found %s)", osArch);
             }
-            if ("ppc64le".equals(osArch)) {
+            if ("ppc64le".equals(osArch) || "aarch64".equals(osArch)) {
                 warnRequirement("Support for the POWER architecture is experimental");
```

Comment:
I think arm and power are different architectures.
```suggestion
                warnRequirement("Support for the POWER and ARM architecture is experimental (found %s)", osArch);
```